### DISCUSSION TITLE
Stats for most searched queries.

### DISCUSSION THREAD

**Discussion by fr3fou** [meilisearch/product#91] [Feedback & Feature Proposal]
It would be really useful to be able to get stats for the most searched queries on a given index. It could be used for very important insights on users' behaivour. Any plans on adding such feature?

**Comment by gmourier** [comment]
Hello @fr3fou! 

Thanks for your feedback. It is currently not planned but it is something we have in mind for the future. We will soon publish a clearer roadmap that is being developed for the post v0.21 era. 😇
